core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXInternalV1Endpoint.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.distrox.api.v1.distrox.endpoint;
+
+import static com.sequenceiq.distrox.api.v1.distrox.doc.DistroXOpDescription.GET_BY_CRN_INTERNAL;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.StackViewV4Response;
+import com.sequenceiq.cloudbreak.doc.Notes;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Path("/v1/internal/distrox")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+@Api(value = "/v1/internal/distrox", protocols = "http,https")
```

Comment:
This will cause a confusion in concept. We also have an internal Endpoint in SDX but it's used for a totally different purpose, to supply custom properties.
Can we come up with a different name?

Reply:
idea is to be internal in CP (not internal as internal user), also we did not find any good name (as I discussed with Ricsi as well)...that can be changed later if someone came up with a better name. But also ... i think SDX internal should be extended with other internal calls if there will be newer ones